### PR TITLE
Copy Worker SSH-Key for root user

### PR DESCRIPTION
**Pull request by RamiGspo**

### CODE REVIEW

File: worker/revpi_factory_reset.sh [hunk 1]
```diff
@@ -27,6 +27,8 @@ ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIDTBh7TFMbVgVDALMMg4VWRO/lnlxy0h4SLlmBxo11PR
 EOX
 
 sshpass -p "$DEFAULT_PASS" ssh-copy-id -i /tmp/lava-key.pub "$DEFAULT_USER"@"$SSH_HOST_RPI"
+ssh pi@revpi32346 "sudo cp -r /home/pi/.ssh /root"
```
Comment (nbuchwitz) [inline worker/revpi_factory_reset.sh]:
```suggestion
sshpass -p "$DEFAULT_PASS" ssh -o StrictHostKeyChecking=no "$DEFAULT_USER"@"$SSH_HOST_RPI" "sudo cp -r /home/pi/.ssh /root"
```

File: worker/revpi_factory_reset.sh [hunk 2]
```diff
@@ -27,6 +27,8 @@ ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIDTBh7TFMbVgVDALMMg4VWRO/lnlxy0h4SLlmBxo11PR
...
+ssh pi@revpi32346 "sudo chown -R root: /root/.ssh"&& rm /tmp/lava-key.pub"
```
Comment (nbuchwitz) [inline worker/revpi_factory_reset.sh]:
```suggestion
sshpass -p "$DEFAULT_PASS" ssh -o StrictHostKeyChecking=no "$DEFAULT_USER"@"$SSH_HOST_RPI" "sudo chown -R root: /root/.ssh"
```

What is the `rm /tmp/lava-key.pub` for? If your intention was to remove the tmp file we created in #4, then please add another commit for this. Also the removal must be run on the worker (eg. after the `ssh-copy-id` line).

**Review by nbuchwitz — CHANGES_REQUESTED** [review]

File: worker/revpi_factory_reset.sh (hunk 2 above)
Comment (RamiGspo) [reply worker/revpi_factory_reset.sh]: We do not need another commit...

**Comment by RamiGspo** [comment]
After making these modifications to the LAVA-worker I have performed several tests and will do some more in the next few days.
Here is some evidence that these changes are working:

Jobs - 2023-05-11-revpi-bullseye-BETA-arm64-lite.img:
[Image installation - 2023-05-11-revpi-bullseye-BETA-arm64-lite.img](https://lava.revpi.kunbus.com/scheduler/job/1394)
[Health check after Image installation - 2023-05-11-revpi-bullseye-BETA-arm64-lite.img](https://lava.revpi.kunbus.com/scheduler/job/1395)
[TestSuite 001 - 2023-05-11-revpi-bullseye-BETA-arm64-lite.img](https://lava.revpi.kunbus.com/scheduler/job/1396)


@nbuchwitz 
I hadn't put it aside or forgotten about it, but in the last 2 days I really haven't had time to do it. I'm sorry for this. Maybe you can still get a good feeling from the tests on your vacation.

**Review by nbuchwitz — APPROVED** [review]

**Pull request merged**